README.md

```diff
@@ -66,6 +68,34 @@ Follow [this guide](./test/spectests/README.md) for using the tool.
 
 This is the unit tests suite of Fizzy.
 
+## Special notes about floating point
+
+### Exceptions
+
+Certain floating point operations can emit *exceptions* as defined by the IEEE 754 standard.
+(Here is a [summary from the GNU C Library](https://www.gnu.org/software/libc/manual/html_node/FP-Exceptions.html)).
+It is however up to the language how these manifest, and in C/C++ depending on the settings they can result in
+a) setting of a flag; b) or in *traps*, such as the `SIGFPE` signal.
+
+Fizzy does not manipulate this setting, but expects it to be left at option a) of above, which is the default.
+In the GNU C Library this can be controlled via the [`feenableexcept` and `fedisableexcept` functions](https://www.gnu.org/savannah-checkouts/gnu/libc/manual/html_node/Control-Functions.html).
+
+The user of the Fizzy library has to ensure this is not set to *trap* mode.
+The behavior with traps enabled is unpredictable and correct behaviour is not guaranteed, thus we strongly advise against using them.
+
+### Rounding
+
+The IEEE 754 standard defines four rounding directions (or modes):
+- to nearest, tie to even (default),
+- toward -∞ (rounding down, floor),
+- toward +∞ (rounding up, ceiling),
+- toward  0 (truncation).
+
+WebAssembly requires "to nearest" rounding, which is the first option, and is the default.
```

Comment:
```suggestion
The semantic of the WebAssembly floating-point instructions is based on the default "no nearest" rounding direction. Therefore, Fizzy expects this rounding direction to be selected. Otherwise, the result of some instructions may be different than what the WebAssembly specification predicts, i.e. the behavior will follow the selected rounding direction.
```

The list of these instructions will be provided later when we have tests in place.

The `floor` instruction should be not on the list, that's why we are having the workaround for the GCC bug already in place.

Reply:
To me this `floor` hack is an inconsistency and I think it should be mentioned, if we want to keep it.

Reply:
Now I see how the hack is consistent with keeping the instructions affected by rounding mode limited to a logical set. Ok to add a list of affected instructions later.